src/css/components/panel.scss

```diff
@@ -252,3 +230,14 @@ h4 + .panel {
   margin: (-$grid-3) (-$grid-3) $grid-3 (-$grid-3);
   padding: $grid-3 $grid-3 $grid-4;
 }
+
+// a couple random panel styles
+
+.route-link {
```

Comment:
Agreed. I'd only done it because there weren't either `routes.scss` or `service-instances.scss` files, and I was hesitant to add them with only a single rule. This was kind of a kludge.

Should I just go ahead and add those two files?